src/eko/evolution_operator/__init__.py

```diff
@@ -159,6 +159,8 @@ def quad_ker(
         coupling value at the process scale
     nf : int
         number of active flavors
+    p : Boolean
+        Polarised (True) or un-Polarised (False)
```

Comment:
Unfortunately I believe that introducing a boolean is the correct way at the moment (or equally bad  to introducing a parallel dispatcher), but eventually I believe we want to get rid of dispatchers at all, at least not keep using them as they are meant now (as a single entry point for Numba).

This will be addressed by refactoring after the analogue of #123 will be implemnted.